browser-extension/background.js

```diff
@@ -26,6 +32,35 @@ chrome.contextMenus.onClicked.addListener(function (e, tab) {
   }
 });
 
+/**
+ * Remove the X-FRAME-OPTIONS and FRAME-OPTIONS headers included in the
+ * Tournesol application HTTP answers. It allows the extension to display
+ * the application in an iframe without enabling all website to do the same.
+ */
+chrome.webRequest.onHeadersReceived.addListener(
+  function(info) {
+    let headers = info.responseHeaders;
+    for (let i=headers.length - 1; i >= 0; --i) {
+      let header = headers[i].name.toLowerCase();
+      if (header == 'x-frame-options' || header == 'frame-options') {
+        headers.splice(i, 1);
+      }
+    }
```

Comment:
Can be made slightly more readable:
```suggestion
    const headers = info.responseHeaders.filter(
      h => !['x-frame-options', 'frame-options'].includes(h.name.toLowerCase())
    )
```